docs/examples/crawl_some_links.ts

```diff
@@ -9,7 +9,7 @@ const crawler = new CheerioCrawler({
         console.log(request.url);
         // Add some links from page to the crawler's RequestQueue
         await enqueueLinks({
-            pseudoUrls: ['http[s?]://apify.com/[.+]/[.+]'],
+            globs: ['apify.com/*/*'],
```

Comment:
@AndreyBykov is this the right way to convert this? It looks fine to me but I don't know if I need to use two `*` or not

Reply:
I think it should be something like `?(http|https)://apify.com/**/*` - but I am not like a huge expert in globs, so would double check it still :/

But in pseudoUrls tests `[http|https]://cool.com/` became `?(http|https)://cool.com/` and `https://example.com/[(\w|-|/)*]` -> `https://example.com/**/*` (and it was verified that it still works the same :) )

Reply:
Maybe we could strip the protocol before we do the matching?

Reply:
Just tried the globs, the most accurate one is `http?://apify.com/*/**`, although if you want be extra strict (so it only matches apify.com/<username>/<actor>`, one `*` per `/` is ideal